src/connectedmachine/README.md

```diff
@@ -2,4 +2,4 @@ Microsoft Azure CLI 'connectedmachine' Extension
 ==========================================
 
 This package is for the 'connectedmachine' extension.
-i.e. 'az connectedmachine'
\ No newline at end of file
+i.e. 'az connectedmachine'
```

Comment:
Can you add more introduction or put links in the README file?

Reply:
updated